xml/System.Windows.Forms/ToolStripControlHost+ToolStripHostedControlAccessibleObject.xml

```diff
@@ -14,7 +14,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Represents the ToolStrip hosted control accessible object which is responsible for accessible navigation within the ToolStrip standard items and hosted controls like TextBox, ComboBox, ProgressBar, etc.</summary>
```

Comment:
Do we need to apply hyperlinks or other forms of formatting to UI controls mentioned here, e.g. `<see cref="T:System.Windows.Forms.ToolStrip" />`?
The same question goes for the items below

Reply:
Probably yes. If you visit the [MS Docs](https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.toolstripcontrolhost.toolstriphostedcontrolaccessibleobject?view=netcore-3.1) page for this type, there aren't any links to `ToolStrip`. My simplified suggestion would be:

```suggestion
    <summary>Represents the accessible object control host responsible for accessible navigation within the <see cref="T:System.Windows.Forms.ToolStrip" /> standard items and hosted controls like TextBox, ComboBox, ProgressBar, etc.</summary>
```

@JuditRose can you please also add the cref hyperlinks to the TextBox, ComboBox, ProgressBar in this description?